install/Dockerfile

```diff
@@ -0,0 +1,78 @@
+# Run "docker build --network=host --tag *** ." to build the image
+
+# Build from nvidia/cuda iamge
+FROM nvidia/cuda:11.0.3-cudnn8-runtime-ubuntu20.04
+SHELL ["/bin/bash", "-c"]
+
+# Add public key of the image to trusted set of keys
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys A4B469963BF863CC
+
+# Change mirrors for apt-get to aliyun 
+RUN sed -i s@/archive.ubuntu.com/@/mirrors.aliyun.com/@g /etc/apt/sources.list
+
+# Install essentials
+RUN DEBIAN_FRONTEND=noninteractive apt-get update\
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y software-properties-common \
+    && DEBIAN_FRONTEND=noninteractive add-apt-repository -y ppa:ubuntu-toolchain-r/test \
+    && DEBIAN_FRONTEND=noninteractive apt-get update \
+    && echo -e "6\n1\n" | DEBIAN_FRONTEND=noninteractive apt-get install -y wget gcc g++ cmake vim libstdc++6
+ENV CC=/usr/bin/gcc
+ENV CXX=/usr/bin/g++
+
+# Install Miniconda package manager.
+#RUN wget -O /root/miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+ADD Miniconda3-latest-Linux-x86_64.sh /root/
+RUN bash /root/Miniconda3-latest-Linux-x86_64.sh -b -p /opt/conda
+RUN rm /root/Miniconda3-latest-Linux-x86_64.sh
+ENV PATH /opt/conda/bin:$PATH
+RUN conda init bash && source /root/.bashrc \
+    && conda config --add channels https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free/  \
+    && conda config --add channels https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main/  \
+    && conda config --add channels https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/pytorch/  \
+    && conda config --add channels https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge  \
+    && conda install python=3.9 libtensorflow_cc=2.6.2=*cuda110* tensorflow=2.6.2=*cuda110* nccl -y
+
+# Solve library inconsistency
+#RUN source activate base && rm ${CONDA_PREFIX}/lib/libtinfo.so* && ln -s /usr/lib/x86_64-linux-gnu/libtinfo.so.6 ${CONDA_PREFIX}/lib/libtinfo.so.6 \
+    #&& rm ${CONDA_PREFIX}/lib/libcurl.so* && ln -s /usr/lib/x86_64-linux-gnu/libcurl.so.4 ${CONDA_PREFIX}/lib/libcurl.so.4
+
+# Compile Plumed from source
+#RUN wget -O ~/plumed-2.8.0.tgz https://github.com/plumed/plumed2/releases/download/v2.8.0/plumed-2.8.0.tgz
+ADD plumed-2.8.0.tgz /root/
```

Comment:
This line is for local installation with a `plumed-2.8.0.tgz` existing at current directory. 
One may have to download a package from GitHub/plumed2 if he doesn't have it. So maybe it's better to use `#RUN wget -O ~/plumed-2.8.0.tgz https://github.com/plumed/plumed2/releases/download/v2.8.0/plumed-2.8.0.tgz` and make `ADD plumed-2.8.0.tgz /root/` comments.